src/store/conversationsStore.js

```diff
@@ -118,16 +117,15 @@ const mutations = {
 	},
 
 	/**
-	 * Patch a conversation object:
-	 * - add new properties
-	 * - update existing properties recursively
-	 * - delete properties
+	 * Update stored conversation if it has changes according to lastActivity
 	 *
 	 * @param {object} state the state
 	 * @param {object} conversation the new conversation object
 	 */
-	patchConversation(state, conversation) {
-		patchObject(state.conversations[conversation.token], conversation)
+	updateConversationIfHasActivity(state, conversation) {
+		if (state.conversations[conversation.token].lastActivity < conversation.lastActivity) {
```

Comment:
I think we also need to do it when the unread counter changes, because reading a chat does not update the lastactivity